src/gasPriceOracle/adapters/ethereum.ts

```diff
@@ -42,10 +46,11 @@ export async function eip1559Raw(
   const maxPriorityFeePerGas = BigNumber.from(_maxPriorityFeePerGas);
   assert(BigNumber.isBigNumber(baseFeePerGas), `No baseFeePerGas received on ${getNetworkName(chainId)}`);
 
+  const scaledPriorityFee = maxPriorityFeePerGas.mul(priorityFeeMultiplier).div(fixedPointAdjustment);
   const scaledBaseFee = baseFeePerGas.mul(baseFeeMultiplier).div(fixedPointAdjustment);
   return {
-    maxFeePerGas: maxPriorityFeePerGas.add(scaledBaseFee),
-    maxPriorityFeePerGas,
+    maxFeePerGas: scaledPriorityFee.add(scaledBaseFee),
+    maxPriorityFeePerGas: scaledPriorityFee,
```

Comment:
We should probably include a comment in the function def that indicates that `maxPriorityFeePerGas` is a scaled value